packages/itwinui-css/src/tile/tile.scss

```diff
@@ -132,6 +128,15 @@
       display: none;
     }
   }
+
+  &.iui-loading {
+    cursor: progress;
+    font-style: italic;
+  }
+
+  &[aria-disabled='true'] {
+    cursor: not-allowed;
+  }
```

Comment:
Is this selector the same as the selector on [L113](https://github.com/iTwin/iTwinUI/pull/765/files#diff-875f9cda040043eba8e927b119436e84c98074bbd862870ed84930dd5265dc25R113)? Or maybe I am missing something

Reply:
Lines 112-126 are the styling for **_both_** loading & disabled states.
Lines 128-131 are the styling for just the loading state.
Lines 133-135 are the styling for just the disabled state.